### PR TITLE
MES-2343 - Added welsh test toggle to office page

### PR DESCRIPTION
**Pull request by roberthall12**
## Description and relevant Jira numbers
Adding the welsh and english radio buttons to the office page. They are pre-populated from the welshTest boolean and allow you to change it during the write up.
https://jira.i-env.net/browse/MES-2343

## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [ ] Any new reusable component/widget added to component library on Confluence

- [ ] Screenshot(s) captured on the physical device (if applicable)
- [ ] Tested by QA
- [ ] PO's approval

Screenshot 2019-05-30 at 16 58 19

### CODE REVIEW

File: src/pages/office/__tests__/office.spec.ts [hunk 1]
```diff
@@ -312,4 +315,12 @@ describe('OfficePage', () => {
       expect(navCtrl.popToRoot).toHaveBeenCalled();
     });
   });
+
+  describe('isWelshChanged', () => {
+    it('should', () => {
```
Comment (jameshighfield) [inline src/pages/office/__tests__/office.spec.ts]: Missing test name here

Comment (roberthall12) [reply src/pages/office/__tests__/office.spec.ts]: Whoops didn't see that, all fixed now

File: src/pages/office/components/language-preference/__tests__/language-preferences.spec.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      .compileComponents()
+      .then(() => {
+        fixture = TestBed.createComponent(LanguagePreferencesComponent);
+        component = fixture.componentInstance;
+        component.formGroup = new FormGroup({});
+      });
+  }));
+
+  describe('Class', () => {
+    it('should compile', () => {
```
Comment (johnmorleybjss) [inline src/pages/office/components/language-preference/__tests__/language-preferences.spec.ts]: Don't bother with this test if you've got other tests that actually do something to the component.

File: src/pages/office/components/language-preference/language-preferences.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <div class="validation-bar"></div>
+    <ion-col class="component-label" col-32 align-self-center>
+      <label>Test language</label>
+    </ion-col>
+    <ion-col align-self-center>
+      <ion-row class="spacing-row"></ion-row>
+      <ion-row>
+        <ion-col col-32>
+          <input type="radio" id="lang-pref-english" class="gds-radio-button" name="language-preferences"
+            formControlName="language-preferences" value=false [checked]="(isWelsh === false)"
```
Comment (johnmorleybjss) [inline src/pages/office/components/language-preference/language-preferences.html]: We generally use camelCase form control names.

**Pull request merged**